packages/ckeditor5-dev-tests/lib/tasks/runmanualtests.js

```diff
@@ -60,7 +61,7 @@ module.exports = function runManualTests( options ) {
 	const language = options.language;
 	const additionalLanguages = options.additionalLanguages;
 	const silent = options.silent || false;
-	const disableWatch = options.disableWatch || false;
+	const disableWatch = options.disableWatch || !isFilesFlagProvided;
```

Comment:
One thing to consider: now it will not be possible to explicitly enable the watch mode if no files are provided.
I'm not sure if this is important as the tests probably will throw an error anyway in that case...

Reply:
Added log that shows up at the end of the script when file watching is disabled.